src/components/popover/popover.tsx

```diff
@@ -11,6 +11,7 @@ import { createRandomString } from '../../util/random-string';
 import { zipObject } from 'lodash-es';
 import { portalContains } from '../portal/contains';
 import { ESCAPE } from '../../util/keycodes';
+import { OpenDirection } from '../menu/menu.types';
```

Comment:
should this be imported from `menu.types';`? or from portal?

Reply:
The types are currently specified in `src/components/menu/menu.types.ts`. That does feel a little bit weird, but it doesn't matter for an external consumer, as they'll just import it with `from '@limetech/lime-elements';` anyway. So we can change it later if we want to.